### PR TITLE
Rearranging the navbar and moving the JWTToken class

### PR DESCRIPTION
**Pull request by JulienMrgrd**
#### 1. In accordance with @jdubois, I reorganized the Registry navbar.

_Before_ :
capture d ecran de 2017-04-28 11-44-15
<br>

_After_ :
capture d ecran de 2017-04-28 11-41-46

<br>
- Tell me if you prefer others icons

See at [fontawesome](http://fontawesome.io/icons/)

#### 2. I moved the JWTToken class (that was in web/rest package).
because all others class inside this package are controllers, and also because it's only used by the UserJWTController class.
Now it's an inner class of web/rest/UserJWTController.java

### CODE REVIEW

File: src/main/java/io/github/jhipster/registry/web/rest/JWTToken.java [hunk 1]
```diff
@@ -1,24 +0,0 @@
-package io.github.jhipster.registry.web.rest;
-
```
Comment (deepu105) [inline src/main/java/io/github/jhipster/registry/web/rest/JWTToken.java]: @JulienMrgrd why was this moved. sorry i merged before i noticed this?

Comment (deepu105) [inline src/main/java/io/github/jhipster/registry/web/rest/JWTToken.java]: @JulienMrgrd please revert this commit alone as this is generated by JHipster so if you wanna change it I suggest you change it in the Generator as well so that its consistent. As i said earlier we need to keep generated code clean as possible so that upgrades are easy

Comment (JulienMrgrd) [inline src/main/java/io/github/jhipster/registry/web/rest/JWTToken.java]: Ok @deepu105, I'll make a PR for the generator soon.
So, you want (on the Registry) that I revert this commit & remake a PR ?

Comment (deepu105) [inline src/main/java/io/github/jhipster/registry/web/rest/JWTToken.java]: you can have it here if you do the same PR in generator :)

Comment (JulienMrgrd) [inline src/main/java/io/github/jhipster/registry/web/rest/JWTToken.java]: Ok, it's better. I'm on it ;)
